src/main/java/seedu/stockpal/commands/ListCommand.java

```diff
@@ -3,9 +3,17 @@
 import seedu.stockpal.data.ProductList;
 import seedu.stockpal.ui.Ui;
 
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static seedu.stockpal.common.Messages.MESSAGE_EMPTY_LIST;
+import static seedu.stockpal.common.Messages.MESSAGE_LIST_SUCCESS;

```

Comment:
Would it be better to import Ui.messages instead of importing the messages to print?